hat/assets/js/apps/Iaso/domains/workflows/config.tsx

```diff
@@ -0,0 +1,190 @@
+    entityTypeId: string,
+    versionId: string,
+): Array<Column> => {
+    const { formatMessage }: { formatMessage: IntlFormatMessage } =
+        useSafeIntl();
+    const columns: Array<Column> = [
+        {
+            Header: formatMessage(MESSAGES.condition),
+            id: 'condition',
+            accessor: 'condition',
+        },
+        {
+            Header: formatMessage(MESSAGES.forms),
+            id: 'forms',
+            sortable: false,
+            Cell: settings => {
+                const { forms } = settings.row.original;
+                return forms.map((form, index) => (
+                    <Box key={form.id}>
+                        <LinkToForm formId={form.id} formName={form.name} />
+                        {index + 1 < forms.length ? ', ' : ''}
+                    </Box>
+                ));
+            },
+        },
+        {
+            Header: formatMessage(MESSAGES.created_at),
+            accessor: 'created_at',
+            sortable: false,
+            id: 'created_at',
+            Cell: DateCell,
+        },
+        {
+            Header: formatMessage(MESSAGES.updated_at),
+            accessor: 'updated_at',
+            sortable: false,
+            id: 'updated_at',
+            Cell: DateCell,
+        },
+        // {
+        //     Header: formatMessage(MESSAGES.actions),
+        //     resizable: false,
+        //     sortable: false,
+        //     accessor: 'actions',
+        //     Cell: settings => {
+        //         return (
+        //             <IconButtonComponent
+        //                 url={`${baseUrls.workflowDetail}/entityTypeId/${entityTypeId}/versionId/${versionId}/followUp/${settings.row.original.id}`}
+        //                 icon="remove-red-eye"
+        //                 tooltipMessage={MESSAGES.see}
+        //             />
+        //         );
+        //     },
+        // },
+    ];
+    return columns;
+};
+
+export const useGetChangesColumns = (
+    entityTypeId: string,
+    versionId: string,
+): Array<Column> => {
+    const { formatMessage }: { formatMessage: IntlFormatMessage } =
+        useSafeIntl();
+    const columns: Array<Column> = [
+        {
+            Header: formatMessage(MESSAGES.form),
+            id: 'form_name',
+            accessor: 'form__name',
+            Cell: settings => {
+                return (
+                    <LinkToForm
+                        formId={settings.row.original.form.id}
+                        formName={settings.row.original.form.name}
+                    />
+                );
+            },
+        },
+        {
+            Header: formatMessage(MESSAGES.mapping),
+            id: 'mapping',
+            accessor: 'mapping',
+            sortable: false,
+            Cell: settings => {
+                return (
+                    <>
+                        {Object.keys(settings.row.original.mapping)
+                            .map(mapKey => mapKey)
```

Comment:
shouldn't we show what the key is mapped to?

Reply:
it was not like that on the mock-ups